frontend/src/actions/send/draw_response_action.ts

```diff
@@ -0,0 +1,20 @@
+import type { ConnectionRepository, Message } from '@/models/connection_repository/connection_repository'
+
+class DrawResponseMessage implements Message {
+    action: string
+    body: {
+        response: boolean
```

Comment:
The naming of the variable `response` is poor. A better option might be `drawAccepted` for example.

Reply:
done